lib/tasks/release.rake

```diff
@@ -3,6 +3,12 @@
 namespace :release do
   desc "Ensures any post-release / pre-deploy behavior has occurred"
   task after_build: [:environment, "db:prepare"] do
+    # @todo Delete after running in prod
+    Marketplace::Order.all.find_each do |order|
+      next unless order.deprecated_delivery_address.present?
+
+      order.update(delivery_address: order.deprecated_delivery_address, deprecated_delivery_address: nil)
+    end
```

Comment:
I see.. this is the prod data migration.....

So.. I assume this rake task runs BEFORE the `delivery_address` column is renamed in `20230302202459_marketplace_encrypt_order_delivery_address.rb`

Reply:
Ah, let me step ya through the Heroku deploy process:

1. Heroku checks out the git repository, and starts a build which creates a new `image` and installs gems, node packages, etc.
2.  Heroku launches a `release` container using the new `image`, and runs the `release` command defined in the `Procfile`, which runs the `rails db:migrate` and release:after_build` tasks in sequence
4. Launches new `web` and `worker` containers using the new `image`.
5. Begins routing traffic to the new `web` containers
6. Stops routing traffic to the old `web` containers
7. Shuts down the old `web` and `worker` containers.

So the database has the new columns prior to running the rake task, *and* before deploying new `web` and `worker` containers. this means that the existing `web` and `worker` containers will still be running the code which references `delivery_address`, which could result in sql query errors. We could [install `strong_migrations`](https://github.com/ankane/strong_migrations) to catch risky operations (like `rename`s); but in this particular case I felt like it was safe enough given our traffic.

That said, there's no reason *not* to use the `Lockbox.migrate` strategy; which eliminates the risk and seems pretty dang cool!

Reply:
I made issue https://github.com/zinc-collective/convene/issues/1177 to install [strong_migrations](https://github.com/ankane/strong_migrations). 

My work team uses it and it is really handy.

Reply:
Yah!!!!